### ISSUE TITLE
Missing splash/screen images for iOS

### ISSUE THREAD

**Issue by freddysu** [quasarframework/app-extension-icon-factory#70]
Seems the following assets for iOS are missed and should be added in ``config.xml`` by quasar-app-extension-icon-factory
```xml
...
<platform name="ios">
  ...
  <splash height="1334" src="res/screen/ios/Default-667h.png" width="750" />
  <splash height="2208" src="res/screen/ios/Default-736h.png" width="1242" />
  <splash height="1242" src="res/screen/ios/Default-Landscape-736h.png" width="2208" />
  <splash height="2436" src="res/screen/ios/Default-2436h.png" width="1125" />
  <splash height="1125" src="res/screen/ios/Default-Landscape-2436h.png" width="2436" />
</platform>
...
```

My Xcode version is 10.2.1 and
``@quasar/quasar-app-extension-icon-factory: "^1.0.0-rc.4"``

Thanks!

**Comment by nothingismagick** [comment]
I’ll do it tomorrow

**Comment by nothingismagick** [comment]
This one is funny:
```
    <splash src="res/screen/ios/Default-667h.png" width="750" height="1334"/>
```
But doing it now. Will be available in 1.0.0
